floris/tools/optimization/yaw_optimization/yaw_optimizer_sr.py

```diff
@@ -240,8 +241,14 @@ def optimize(self, print_progress=True):
                 # Evaluate grid of yaw angles, get farm powers and find optimal solutions
                 farm_powers = self._process_evaluation_grid()
 
+                # If farm powers contains any nans, then issue a warning
+                if np.any(np.isnan(farm_powers)):
+                    warnings.warn(
+                        "WARNING: NaNs found in farm powers. SR using max non-NaN "
```

Comment:
This warning message is in my opinion a little cryptic and it would be good to add some clarity around what it means that there are NaNs. I propose something like this:
"SerialRefine optimization has NaNs in farm powers; the optimization will ignore yaw settings with invalid power and proceed"

Reply:
The idea for adding more context is that we have a few different messages coming at the user. On its own and outside of the context of this file, this message may be ambiguous. It would be helpful to readers to have more context that could guide them to look in particular places for an issue or more information.